src/Nethermind/Nethermind.Consensus/Transactions/MinGasPriceTxFilter.cs

```diff
@@ -48,7 +48,8 @@ public AcceptTxResult IsAllowed(Transaction tx, BlockHeader? parentHeader, in UI
             UInt256 premiumPerGas = tx.GasPrice;
             UInt256 baseFeePerGas = UInt256.Zero;
             long blockNumber = (parentHeader?.Number ?? 0) + 1;
-            IReleaseSpec spec = _specProvider.GetSpec(blockNumber);
+            ulong blockTimestamp = (parentHeader?.Timestamp ?? 0) + 13;
```

Comment:
13 should come from some slot time value? It is different for Gnosis. BTW we have that in MergeConfig already.